package.xml

```diff
@@ -20,7 +20,7 @@
 
   <license>Apache License, Version 2.0</license>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
```

Comment:
Please update the top comment
ROS --> ROS2
via catkin_make --> via ament_cmake

Reply:
Done